pkg/cmd/cli/cli_changefeed_create.go

```diff
@@ -187,12 +188,27 @@ func (o *createChangefeedOptions) completeReplicaCfg() error {
 // validate checks that the provided attach options are specified.
 func (o *createChangefeedOptions) validate(cmd *cobra.Command) error {
 	if o.timezone != "SYSTEM" {
-		cmd.Printf(color.HiYellowString("[WARN] --tz is deprecated in changefeed settings.\n"))
+		cmd.Println(color.HiYellowString("[WARN] --tz is deprecated in changefeed settings.\n"))
+	}
+
+	if len(o.commonChangefeedOptions.configFile) > 0 {
+		cfg := config.GetDefaultReplicaConfig()
+		metaData, err := toml.DecodeFile(o.commonChangefeedOptions.configFile, cfg)
+		if err != nil {
+			return errors.Trace(err)
+		}
+
+		if metaData.IsDefined("sink", "enable-partition-separator") {
```

Comment:
Can we ensure it has no effect when set to false?  Since this could potentially lead to data inconsistency.

Reply:
Yes, but we decide to change the behavior in later versions. You can check the comments in the issue.